docker/suricata/dist/suricata.yaml

```diff
@@ -749,23 +756,26 @@ app-layer:
     ftp:
       enabled: yes
       # memcap: 64mb
-    # RDP, disabled by default.
     rdp:
       enabled: yes
     ssh:
       enabled: yes
+      hassh: yes
```

Comment:
Not sure how this plays with the FATT honeypot.

Reply:
Worked flawlessly so far since both reside in differrent containers.